### ISSUE TITLE
Notification received while app is running kills the app

### ISSUE THREAD

**Issue by JueBag** [openhab/openhab-android#853]
1m
Just tested the restarted notification service and found a problem.
If the app is running in background while the message is received followed by the message clicked OK, the app gets either stopped or a selection window gets displayed asking for stopping the app or giving fedback (checking on two different android devices)

App Version 2.1.0

**Comment by maniac103** [comment]
Can you please show a screenshot of what you mean by 'the message clicked OK'? Thanks.

**Comment by JueBag** [comment]
![stopped](https://user-images.githubusercontent.com/18365158/39092944-a933c390-4617-11e8-8673-4fb015f5aa46.png)
![message](https://user-images.githubusercontent.com/18365158/39092945-b30dc2f8-4617-11e8-9f7c-31b6831e6f7f.png)
Selecting the received message from the second picture results in the displayed message in the first picture.
This message made me think that the app was stopped, however it seems to remain working. Sorry for that "false" report, however I think the displayed message telling that the app is stopped shouldn't come up.

**Comment by maniac103** [comment]
Indeed, it shouldn't come up. You can't collect logcat after that incidence by chance, can you?
@mueller-ma Is there any ANR report in the Play Store console?

**Comment by mueller-ma** [comment]
@JueBag Which device and Android version do you use?

@maniac103 There is only one ANR for the 2.1.0 version:
````
Input dispatching timed out (Waiting to send non-key event because the touched window has not finished processing certain input events that were delivered to it over 500.0ms ago. Wait queue length: 92. Wait queue head age: 860185.5ms.)
````
There are a lot of stacktraces show for this anr, but none of them contain lines about the org.openhab.habdroid package.

**Comment by maniac103** [comment]
@mueller-ma May I get read-only access to the Play Store console so I can check the crashes myself? If yes, how can I non-publically tell you my Gmail address?

**Comment by JueBag** [comment]
Got LogCat,  recorded while sending a message from the webpage. However I can't see anything in that log, except maybe an entry "Input channel destroyed" (but I'm new to this.)

**Comment by mueller-ma** [comment]
> May I get read-only access to the Play Store console so I can check the crashes myself? If yes, how can I non-publically tell you my Gmail address?

While it makes sense to give you access to stacktraces, @kaikreuzer needs to decide that.

**Comment by JueBag** [comment]
Yes,  my devices are not rooted. Since I'm away from my PC (Scotland atm) I can't help on that topic. Sorry

**Comment by maniac103** [comment]
I can reproduce, so never mind :-) I've seen that particular issue before, and thought I fixed it at some point, but the fix probably got lost during rebasing or in some refactor. I'll fix it.

**Comment by maniac103** [comment]
https://github.com/openhab/openhab-android/commit/9c683b20ca620a21ece5e34ed48dc61e1ce253a1 is the commit I thought of. It didn't get lost, but is not sufficient either: it should check for 'cloud connection present && started' before opening the notifications pane.

**Comment by mueller-ma** [comment]
@maniac103 You should have received a mail from me.
